modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMDataProviderInstanceServiceImpl.java

```diff
@@ -240,11 +240,6 @@ public DDMDataProviderInstance updateDataProviderInstance(
 			ddmFormValues, serviceContext);
 	}
 
-	@Reference(unbind = "-")
-	protected void setJSONFactory(JSONFactory jsonFactory) {
```

Comment:
@JanisZhang For this kind of removal, please write the commit message like "Remove redundant method reference in xxx as the field is already referenced".

Reply:
@dantewang Got it,thanks!